src/Algebra/Definitions/RawMagma.agda

```diff
@@ -54,15 +56,26 @@ _∤_ : Rel A (a ⊔ ℓ)
 x ∤ y = ¬ x ∣ y
 
 ------------------------------------------------------------------------
--- Mutual divisibility
-
--- When in a cancellative monoid, elements related by _∣∣_ are called
--- associated and if x ∣∣ y then x and y differ by an invertible factor.
-
-infix 5 _∣∣_ _∤∤_
+-- Mutual divisibility _∣∣_.
+-- In a  monoid,  this is an equivalence relation extending _≈_.
+-- When in a  cancellative monoid,  elements related by _∣∣_ are called
+-- associated,  and there x ∣∣ y is equivalent to that x and y differ by
+-- some invertible factor.
+-- Example: for ℕ  this is equivalent to  x ≡ y,
+--          for ℤ  this is equivalent to  (x ≡ y  or  x ≡ -y).
 
 _∣∣_ : Rel A (a ⊔ ℓ)
 x ∣∣ y = x ∣ y × y ∣ x
 
 _∤∤_ : Rel A (a ⊔ ℓ)
-x ∤∤ y =  ¬ x ∣∣ y
+x ∤∤ y = ¬ x ∣∣ y
+
+------------------------------------------------------------------------
+-- Greatest common divisor (GCD)
+
+record IsGCD (x y gcd : A) : Set (a ⊔ ℓ) where
```

Comment:
You've now added two copies of this definition, one in `Algebra.Definitions.RawMagma` and one in `Algebra.Definitions.RawSemiring` :cry: